ConsentViewController/Classes/GDPRConsentViewController.swift

```diff
@@ -374,15 +374,6 @@ extension GDPRConsentViewController: GDPRConsentDelegate {
                     self?.onError(error: error)
                 }
             }
-        } else if (action.type == .Dismiss) {
```

Comment:
@vilasma why was this code deleted? If the message is dismissed, will the `onConsentReady` still be called?

Reply:
@andresilveirah Yes, I moved the dismiss action call to MessageWebViewController.